docs/api/rest_api/rest_api_usage/rest_requests.md

```diff
@@ -136,6 +137,14 @@ Examples of such requests are:
 - [moving a Location and its subtree](../rest_api_reference/rest_api_reference.html#managing-content-move-subtree)
 - [swapping a Location with another](../rest_api_reference/rest_api_reference.html#managing-content-swap-location)
 
+### Expected user
+
+The `X-Expected-User` header specifies the user needed on client side for the request execution.
+If the current username on server side isn't equal to `X-Expected-User` value, a `401 Unauthorized` error is returned.
```

Comment:
A sentence explaining behavior without the `X-Expected-User` header would in my opinion be helpful here.

Defaulting to anonymous user is implied in the next paragraph, but it isn't explicitly said.

Reply:
@Steveb-p What about this?:
```suggestion
With this header, if the current username on server side isn't equal to `X-Expected-User` value, a `401 Unauthorized` error is returned.
Without this header, the request will be executed with the current user who might be unexpected (like the Anonymous user if a previous authentication has expired) and an ambiguous response might be returned without a clue that the request was a success but with a wrong user.
```

Reply:
Sounds good to me :+1: